node/src/command.rs

```diff
@@ -394,106 +422,129 @@ pub fn run() -> Result<()> {
 							<PendulumRuntimeExecutor as NativeExecutionDispatch>::ExtendHostFunctions,
 						>, _>(Some(substrate_info(BLOCK_TIME_MILLIS))), task_manager))
 					}),
+					ChainIdentity::FoucocoStandalone => todo!(),
 				}
 			} else {
 				Err("Try-runtime must be enabled by `--features try-runtime`.".into())
 			}
 		},
+
 		None => {
 			let runner = cli.create_runner(&cli.run.normalize())?;
-			let collator_options = cli.run.collator_options();
-
-			runner.run_node_until_exit(|config| async move {
-				let hwbench = if !cli.no_hardware_benchmarks {
-					config.database.path().map(|database_path| {
-						let _ = std::fs::create_dir_all(database_path);
-						sc_sysinfo::gather_hwbench(Some(database_path))
-					})
-				} else {
-					None
-				};
-
-				let para_id = chain_spec::ParachainExtensions::try_get(&*config.chain_spec)
-					.map(|e| e.para_id)
-					.ok_or("Could not find parachain ID in chain-spec.")?;
-
-				let polkadot_cli = RelayChainCli::new(
-					&config,
-					[RelayChainCli::executable_name()].iter().chain(cli.relay_chain_args.iter()),
-				);
-
-				let id = ParaId::from(para_id);
-
-				let parachain_account =
-					AccountIdConversion::<polkadot_primitives::v4::AccountId>::into_account_truncating(&id);
-
-				let state_version = Cli::native_runtime_version(&config.chain_spec).state_version();
-				let block: Block = generate_genesis_block(&*config.chain_spec, state_version)
-					.map_err(|e| format!("{e:?}"))?;
-				let genesis_state = format!("0x{:?}", HexDisplay::from(&block.header().encode()));
-
-				let tokio_handle = config.tokio_handle.clone();
-				let polkadot_config =
-					SubstrateCli::create_configuration(&polkadot_cli, &polkadot_cli, tokio_handle)
-						.map_err(|err| format!("Relay chain argument error: {err}"))?;
+			runner
+				.run_node_until_exit(|config| async move {
+					if cli.instant_seal {
+						start_instant(config).await
```

Comment:
I don't understand what all the code in `start_node` is doing. Sure that we need none of this for the foucoco-standalone version?

Reply:
As far as I am aware it is mostly to start the relay configurations, which we don't use (and in fact we don't want) in the standalone [version](https://github.com/pendulum-chain/foucoco-standalone/blob/main/node/src/command.rs#L122).

Reply:
To better understand this: is the new foucoco-standalone runtime you are defining here a standalone substrate chain or a parachain (i.e., it also requires a relay chain?)

Reply:
It does not require a relay chain and works like the standalone version in the repo (foucoco-standalone repo).

Reply:
But then wouldn't we need to define a completely different runtime with a different set of pallets? Don't the cumulus pallets require a connection to a relay chain, etc?

Reply:
At least during testing I was able to run the deployment of the nabla contracts, I believe it ignores the runtime logic that relates to the real consensus mechanisms (cumulus pallets, for example). I thought @ebma was referring just to the name(?), but this is an important point. 

Do you think testing/deploying `nabla` (or any `wasm-deploy` script, etc) is not enough to ensure that the runtime logic works on this standalone version, even with those pallets?

Reply:
If the standard foucoco runtime (that is intended to be a parachain) really happens to work as a standalone chain, then that's awesome and should be enough for the Nabla tests. I seriously did not expect that to work. Did you try to run the Nabla test suite with it?

Reply:
Interesting, I honestly assumed that the new instant seal runtime would run like the others. If it's running like a standalone chain, there is no need to rename it after all. Seems like the best of both worlds for us, except the overhead of maintaining the extra logic in the node but we probably cannot avoid that. 

We would need to ask ourselves if it makes any difference to our testing setup. In the past, we noticed that the block weight limits were different between the standalone chain and our parachains. I didn't see any changes related to this in the code so I would now naively assume that we don't face this issue going forward as the configuration is shared between standalone and parachain runtime. Anything else that comes to mind that we should pay attention to @gianfra-t or @TorstenStueber?

Reply:
Yes it was a pleasant surprise that the runtime worked without modifications. When I ran the Nabla test suite I encountered the known "contract too long" error `CodeTooLarge`, so I couldn't verify completely.

But deploying also calls the chain extensions in order to approve so I have reason to believe all the components work. The only problem I've encountered is because the db gets corrupted or conflicts with a previous run, I don't know how this was handled before but to solve I either delete it or change the id of the chain temporarily (which creates a new db). 

@ebma am not sure about weight limits, but the configurations must now the same. It is hard to think of another issue that could arise.

Reply:
>  The only problem I've encountered is because the db gets corrupted or conflicts with a previous run, I don't know how this was handled before but to solve I either delete it or change the id of the chain temporarily (which creates a new db).

How exactly did you encounter this issue?

> I  am not sure about weight limits, but the configurations must now the same

Yeah I guess we can assume they are the same 👍

Reply:
@ebma It could be hard to research whether there are some subtle differences. For now at least it might be enough if we can use it for Nabla (or similar smart contracts). Do you see other uses for the standalone version?

The "contract too long" error is a problem to run the test suit but it should be unrelated to foucoco-standalone.

I don't know how you mean that the db gets corrupted. When I run the test suite, I always start everything from scratch: foucoco-standalone starting at genesis and the indexer running locally with an empty db.

Reply:
About the db corruption: Since while testing I started the node many times, I found two strange errors: the node crashing on star and the deployment of the contracts not being successful due to failure in a chain extension call. 

For the first case, it happened that at the beginning when I was using the same config name `foucoco` for the standalone version, this meant that upon start the node would use the already existing db used for `foucoco`. Now, I don't know why exactly this happened, but simply changing the [name](https://github.com/pendulum-chain/pendulum/pull/462/files/1607e16b522fbe71946be7e67dd063a0d484616e#diff-f2256017c4e606a30f53708578da46675512cc1300be6542ed5f77e58af0d6dcR264) to, for example, `foucoco2` would solve the issue. This happened only these two times and I couldn't replicate. I suspect it was the db since it was the only variable.